CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -1,6 +1,35 @@
 
 <hr>
 
+## v1.4.0-alpha.1(TBD)
+
+### BoltDB
+- [Enhance check functionality to support checking starting from a pageId](https://github.com/etcd-io/bbolt/pull/659)
+- [Optimize the logger performance for frequent called methods](https://github.com/etcd-io/bbolt/pull/741)
+- [Stabilize the behaviour of Prev when the cursor already points to the first element](https://github.com/etcd-io/bbolt/pull/734)
+
+### CMD
+- [Fix `bbolt keys` and `bbolt get` to prevent them from panicking when no parameter provided](https://github.com/etcd-io/bbolt/pull/682)
+- [Fix surgery freelist command in info logs](https://github.com/etcd-io/bbolt/pull/700)
+- [remove txid references in surgery meta command's comment and description](https://github.com/etcd-io/bbolt/pull/703)
```

Comment:
Suggest to capitalize sentence to keep it consistent with the other lines:

```suggestion
- [Remove txid references in surgery meta command's comment and description](https://github.com/etcd-io/bbolt/pull/703)
```

Reply:
Updated & thx